ssz-rs/src/de.rs

```diff
@@ -48,6 +55,8 @@ impl Display for DeserializeError {
             ),
             DeserializeError::InvalidInstance(err) => write!(f, "invalid instance: {err}"),
             DeserializeError::InvalidType(err) => write!(f, "invalid type: {err}"),
+            DeserializeError::IncompleteLengthOffsets(err) => write!(f, "incomplete length offsets: {err}"),
+            DeserializeError::OffsetNotIncreasing { start, end } => write!(f, "empty span from {start} to {end} bytes"),
```

Comment:
I think this is a little clearer
```suggestion
            DeserializeError::OffsetNotIncreasing { start, end } => write!(f, "offset points to {end} bytes before {start} bytes"),
```